pages/clustering/high-availability.mdx

```diff
@@ -382,9 +390,9 @@ are all disabled.
 
 <Callout type="warning">
 
-It is important to note that our current implementation of high availability could lead to data loss. The specific situation could occur when all REPLICAs are behind the current MAIN
-and current MAIN is down. The failover is triggered in which the most-up-to-date replica is chosen for promotion. Such an incident can only happen in high-volume environment where
-data is constantly changed, added, deleted etc. If zero data-loss HA is your requirement, please open an issue on [GitHub](https://github.com/memgraph/memgraph/issues) and we will work towards improving that.
+Under certain extreme scenarios, the current implementation of HA could lead to data loss. These are scenarios of high-volume environment 
+where data is constantly changed, added, deleted... If you are operating in such scenarios, please open an issue on [GitHub](https://github.com/memgraph/memgraph/issues) 
+as we would love to help you out.
```

Comment:
This might be too informal. What do you think about:
`as we are eager to expand our support for this kind of workload.`
?